BUILD_MESON

```diff
@@ -18,6 +18,20 @@ Buidling Dinit is available via two "Build system": meson & make.
 Its file tells everything you need to buidling Dinit via meson.
```

Comment:
Should (probably) say:
"This file contains instructions for building Dinit using meson."